generic_config_updater/generic_updater.py

```diff
@@ -29,16 +30,18 @@ def __init__(self,
                  patchsorter=None,
                  changeapplier=None,
                  config_wrapper=None,
-                 patch_wrapper=None):
+                 patch_wrapper=None,
+                 namespace=multi_asic.DEFAULT_NAMESPACE):
+        self.namespace = namespace
         self.logger = genericUpdaterLogging.get_logger(title="Patch Applier", print_all_to_console=True)
-        self.config_wrapper = config_wrapper if config_wrapper is not None else ConfigWrapper()
-        self.patch_wrapper = patch_wrapper if patch_wrapper is not None else PatchWrapper()
+        self.config_wrapper = config_wrapper if config_wrapper is not None else ConfigWrapper(namespace=self.namespace)
+        self.patch_wrapper = patch_wrapper if patch_wrapper is not None else PatchWrapper(namespace=self.namespace)
         self.patchsorter = patchsorter if patchsorter is not None else StrictPatchSorter(self.config_wrapper, self.patch_wrapper)
-        self.changeapplier = changeapplier if changeapplier is not None else ChangeApplier()
+        self.changeapplier = changeapplier if changeapplier is not None else ChangeApplier(namespace=self.namespace)
 
     def apply(self, patch, sort=True):
-        self.logger.log_notice("Patch application starting.")
-        self.logger.log_notice(f"Patch: {patch}")
+        self.logger.log_notice(f"{self.namespace}: Patch application starting.")
```

Comment:
when `namespace=multi_asic.DEFAULT_NAMESPACE` can we print `localhost` iso `''`?

Reply:
Nice finding!